src/pvecontrol/utils.py

```diff
@@ -30,10 +31,13 @@ def print_taskstatus(task):
   output = [ filter_keys(task.__dict__, ['upid', 'exitstatus', 'node', 'runningstatus', 'type', 'user', 'starttime']) ]
   print_tableoutput(output)
 
-def print_task(proxmox, upid, follow = False):
+def print_task(proxmox, upid, follow = False, wait = False, show_logs = False):
   task = proxmox.find_task(upid)
   logging.debug("Task: %s", task)
-  print_taskstatus(task)
+
+  if task.running() and (follow or wait):
```

Comment:
I have a doubt about what it does.
IMO `--wait` should just create a process lock until task status is running or failed (it could return 0 or 1).
But `--wait` should not print anything by default ? `--follow` seems to be a `--wait` with a `show_log` true.